tests/sentry/models/test_apiapplication.py

```diff
@@ -13,15 +13,15 @@ def test_is_valid_redirect_uri(self):
         assert app.is_valid_redirect_uri("http://example.com/")
         assert app.is_valid_redirect_uri("http://example.com")
         assert app.is_valid_redirect_uri("http://example.com/.")
-        assert app.is_valid_redirect_uri("http://example.com//")
-        assert app.is_valid_redirect_uri("http://example.com/biz/baz")
+        assert not app.is_valid_redirect_uri("http://example.com//")
```

Comment:
First three examples are normalized to `/` by user agents. However, the last one is `//`, considered a different URL:
```
$ curl -v http://example.com/ 2>&1 | grep GET
> GET / HTTP/1.1
$ curl -v http://example.com 2>&1 | grep GET
> GET / HTTP/1.1
$ curl -v http://example.com/. 2>&1 | grep GET
> GET / HTTP/1.1
$ curl -v http://example.com// 2>&1 | grep GET
> GET // HTTP/1.1
```